content/components/action-bar.mdx

```diff
@@ -1,84 +1,89 @@
 ---
 title: Action bar
 status: Experimental
-description: An action bar contains a collection of horizontally aligned icon buttons.
+description: An action bar contains a collection of horizontally aligned actions.
 ---
 
 import {Box, Text} from '@primer/react'
 
-## Overview
-
-Use an action bar to render multiple icon buttons in a row. Buttons can be split into groups by adding a divider. When there is not enough space, buttons that don't fit will be added to an overflow menu.
+<img
+  width="960"
+  alt=""
+  src="https://user-images.githubusercontent.com/378023/197716033-86fdac40-2b14-438c-a712-b003b76dabd3.png"
+/>
 
 ## Anatomy
 
+And action bar is mostly a wrapper for different kind of actions. It makes sure alliment and responsiveness works without much configurations.
+
 <img
   width="960"
   alt="A diagram of an action bar with a few buttons, a divider and at the end a button that opens an overflow menu."
-  src="https://user-images.githubusercontent.com/378023/193506131-be8bead4-d1ab-4c79-8ac1-054e75bac9be.png"
+  src="https://user-images.githubusercontent.com/378023/198211386-3e48eeb4-8cc6-41c0-85dd-1596aaf0d3ab.png"
 />
 
-## Content
+### Items
 
-### Buttons
+<Box
+  as="figure"
+  display={["block", "block", "block", "block", "grid"]}
+  gridTemplateColumns="1fr 1fr"
+  gridGap={7}
+  m={0}
+  mb={3}
+>
+  <Text as="p" mt="0">
+Each Action bar item is componsed of a visual and a label.
```

Comment:
```suggestion
Each action bar item is composed of a visual and/or a label.
```

It could have no visual and just text or no label, right?

Reply:
Hmm.. this is tricky. Each item always needs a label **AND** a visual. But it's true that sometimes the label is only used for the tooltip or the visual is only used when inside the overflow menu. I'll change it to `requires`. Then it's not tied to being visible at all times.